src/panel/app_panel.js

```diff
@@ -21,24 +22,28 @@ import Router from 'src/proxies/app_router';
 import Poi from 'src/adapters/poi/poi.js';
 import layouts from './layouts.js';
 import ReactPanelWrapper from 'src/panel/reactPanelWrapper';
+import events from '../../config/events.yml';
```

Comment:
Just a note that you can also write `config/events.yml` as there is an alias defined.